k8s.gcr.io/images/k8s-staging-provider-azure/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+# Should always mirror the maintainers/reviewers in https://sigs.k8s.io/cluster-api-provider-azure/OWNERS_ALIASES
+
+approvers:
```

Comment:
This list needs to include everyone from https://github.com/kubernetes/cloud-provider-azure/blob/master/OWNERS.